packages/app-store/googlecalendar/tests/google-calendar.e2e.ts

```diff
@@ -18,9 +18,13 @@ test.describe("Google Calendar", async () => {
     let qaGCalCredential: Prisma.CredentialGetPayload<{ select: { id: true } }>;
     test.beforeAll(async () => {
       let runIntegrationTest = false;
+      let errorMessage = "Could not run test";
 
       test.skip(!!APP_CREDENTIAL_SHARING_ENABLED, "Credential sharing enabled");
 
+      test.skip(!process.env.E2E_TEST_CALCOM_QA_EMAIL, "QA email not found");
+      test.skip(!process.env.E2E_TEST_CALCOM_QA_PASSWORD, "QA password not found");
```

Comment:
Lets add the lint rules to these lines to prevent these eslint warnings creeping in  <3